src/runtime_src/core/tools/common/ReportElectrical.cpp

```diff
@@ -66,7 +80,15 @@ ReportElectrical::getPropertyTree20202( const xrt_core::device * _pDevice,
                                            boost::property_tree::ptree &_pt) const
 {
   boost::property_tree::ptree pt;
-  
+  std::string power_watts;
+  try {
+    auto power_level = xrt_core::device_query<qr::max_power_level>(_pDevice);
+    power_watts = lvl_to_power_watts(power_level);
+  }
+  catch (...) {
+    power_watts = "N/A";

```

Comment:
Hmm...there seems to be some inconsistencies here. 

If an exception is thrown, power_watts is "N/A", but if the power level value is greater then 3 it has a value of zero.  Shouldn't they both be same value possible "N/A"